### PR TITLE
Support showing Players in the frontend

### PR DESCRIPTION
**Pull request by Omustardo**

### CODE REVIEW

**Review by thirdknife — APPROVED** [review]

File: cmd/main.go [hunk 1]
```diff
@@ -57,7 +73,14 @@ func main() {
 	})
 
 	e.GET("/players", func(c echo.Context) error {
-		return c.Render(http.StatusOK, "players", nil)
+		var players []*database.Player
+		result := db.Find(&players) // SELECT * FROM Players;
+		if result.Error != nil {
+			fmt.Errorf("Error loading players: %v", result.Error)
+			return c.HTML(http.StatusInternalServerError, "<p>Error fetching players.</p>")
```
Comment (thirdknife) [inline cmd/main.go:78]: did this actually work?

File: cmd/main.go [hunk 2]
```diff
@@ -57,7 +73,14 @@ func main() {
 	})
 
 	e.GET("/players", func(c echo.Context) error {
-		return c.Render(http.StatusOK, "players", nil)
+		var players []*database.Player
+		result := db.Find(&players) // SELECT * FROM Players;
```
Comment (thirdknife) [inline cmd/main.go]: can you move this out to a another layer?

File: cmd/main.go (hunk 1 above)
Comment (Omustardo) [reply cmd/main.go:78]: Yep - HTML at the page is just the content provided here, so just:
`<p>Error fetching players.</p>`

File: cmd/main.go (hunk 2 above)
Comment (Omustardo) [reply cmd/main.go]: Sure, done.

File: cmd/main.go (hunk 1 above)
Comment (Omustardo) [reply cmd/main.go:78]: Eventually we'll want to have htmx templates that display error messages inline, or something nicer than this, but this does technically work.

**Pull request merged**